src/main/java/com/google/sps/servlets/BooksServlet.java

```diff
@@ -44,10 +44,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     List<String> terms = Arrays.asList(request.getParameterValues("key"));
     LinkedHashMap<String, List<Book>> booksMap = new LinkedHashMap<>();
     terms.forEach((term) -> {
-      // String jsonString = getJsonStringForTerm(term);
-      // List<Book> books = makeBooksList(jsonString);
-      // booksMap.put(term, books);
-      booksMap.put(term, Collections.emptyList());
+      try {
+        String jsonString = getJsonStringForTerm(term);
+        List<Book> books = makeBooksList(jsonString);
+        booksMap.put(term, books);
+      } catch(Exception e) {
```

Comment:
What kind of Exceptions might occur here?  Can we limit to catching the specific type of Exception we expect?

Reply:
This was marked as resolved but I don't see a response.  Can you please clarify?

Reply:
Oh sorry, I fixed it in ProjectsServlet.java,.